bonus_guides/D_mix_tasks.md

```diff
@@ -810,3 +810,14 @@ Greetings from the Hello Phoenix Application!
 ```
 
 Indeed it does.
+
+If you want to make your new mix task to use your application's infrastructure, you need to make sure the application is started when mix task is being executed. This is particulary useful if you need to access your database from within the mix task. Thankfully, mix makes it really easy for us:
+
+```elixir
+  . . .
+  def run(_args) do
+    Mix.Task.run "app.start"
```

Comment:
I'm not sure we want to start the whole app though? Maybe instead we could start the repo directly? `Repo.start_link`